apis/apis.md

```diff
@@ -5,12 +5,14 @@ title: APIs
 description: Machine translation APIs
 ---
 
-There is a growing set of machine translation engines available.
+There is a growing set of machine translation APIs available.
 
-The most popular engines are available as self-serve APIs and support more than 100 [languages](/languages/languages.md) and more than 10000 language pairs.
+The most popular APIs are available as self-serve APIs and support more than 100 [languages](/languages/languages.md) and more than 10000 language pairs.  By 2022, there was a self-serve API that supported more than 300 languages.
 
-By 2022, there was a self-serve API that supported more than 300 languages.
+Besides language and locale support, the APIs also vary by features like customisation, privacy policies and whether or not they are self-service products.  Many of the APIs have been integrated into translation management systems (TMS) where human translators work to [post-edit](https://machinetranslate.org/post-editing) machine translation output.
```

Comment:
Let's link to the locale article, and perhaps privacy policies ([this article](/industry/data-confidentiality.md)) and products?

Reply:
That's a different type of product